sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -452,8 +457,7 @@ def cleanup(
 # Examples:
 # ib.clusters.describe(p)
 # Check the docstrings for detailed usages.
-# TODO(victorhc): Implement all functionality for Clusters()
-# clusters = Clusters()
+clusters = Clusters()
```

Comment:
Let's not enable it until we have a working example. You can update the TODO item to be "Resolve connection issue and add a working example"

Reply:
Sounds good, I reverted this and changed the comment. Thanks!